src/components/Days/DayCard/DayCard.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import logo from '../../../assets/sun.svg';
+import classes from './DayCard.module.scss';
+
+export const DayCard = (props) => {
+  let cardClasses = [classes.Card];
```

Comment:
In this case a 'const' is more suitable because the cardClasses variable is never reassigned.